app/src/activity.c

```diff
@@ -74,8 +74,8 @@ void activity_work_handler(struct k_work *work) {
     } else
 #endif /* IS_ENABLED(CONFIG_ZMK_SLEEP) */
         if (inactive_time > MAX_IDLE_MS) {
```

Comment:
Minor: weird indentation change here. Should this line be un-indented one level too?

Reply:
clang-format is doing that consistently... The preprocessor directive is maybe ignored by clang-format?

Reply:
`else if` on different lines would have different levels of indentation.